airflow/models/skipmixin.py

```diff
@@ -17,7 +17,9 @@
 # under the License.
 
 import warnings
-from typing import TYPE_CHECKING, Iterable, Union
+from typing import TYPE_CHECKING, Iterable, Optional, Sequence, Union
+
+import pendulum
```

Comment:
Since we already have `if TYPE_CHECKING` on L30 move this to that block?

Reply:
Moved